requirements/edx/github.txt

```diff
@@ -31,6 +31,7 @@ git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c4aee6e76b9abe61cc808
 # custom opaque-key implementations for ccx
 -e git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
+-e git+https://github.com/solashirai/crowdsourcehinter.git@b92d9fae317550b6942f201a9338e2bee3720064#egg=crowdsourcehinter-xblock
```

Comment:
> a malicious user would need to have access to the repo

That's absolutely correct. Right now, we will do a review of Sola's code (or the RecommenderXBlock or ...). It is known to not contain back doors when it merges. Whenever we upgrade the hash, someone looks at the delta. 

If we switched to tags, a malicious user would just need to compromise the account of *any* developer who has push access to *any* repo included in any of our requirements files to 0wn our servers. That's a very large security perimeter. 

If a hash is rebased or garbage collected out of existence, the behavior we should see is a build a failure. This is a *good* behavior. If the code we want to deploy no longer exists, the servers should not build. With tags, we see a phantom change of code versions on our servers. It still builds, but it behaves differently, and possibly incorrectly. There's basically no way to debug or trace down why. That's a very bad behavior. @e0d